docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -0,0 +1,212 @@
+import nvidia.dali as dali
+
+def input(file_root, annotations_file, device_id, num_threads, device, random_shuffle=True):
+    inputs, bboxes, classes = dali.fn.readers.coco(
+        file_root=file_root,
+        annotations_file=annotations_file,
+        ltrb=True,
+        shard_id=device_id,
+        num_shards=num_threads,
+        ratio=True,
+        random_shuffle=random_shuffle
+    )
+    images = dali.fn.decoders.image(inputs, device=device, output_type=dali.types.RGB)
+
+    return images, bboxes, classes
+
+def ltrb_to_xywh(bboxes):
+    Z = dali.types.Constant(0.0)
+    H = dali.types.Constant(0.5)
+    O = dali.types.Constant(1.0)
+    MO = dali.types.Constant(-1.0)
+
+    M = dali.fn.stack(
+            dali.fn.stack(H, Z, H, Z),
+            dali.fn.stack(Z, H, Z, H),
+            dali.fn.stack(MO, Z, O, Z),
+            dali.fn.stack(Z, MO, Z, O)
+        )
+    return dali.fn.coord_transform(bboxes, M=M)
+
+
+def bbox_adjust_xywh(bboxes, shape_x, shape_y, pos_x, pos_y):
+    Z = dali.types.Constant(0.0)
+
+    M = dali.fn.stack(
+            dali.fn.stack(shape_x, Z, Z, Z),
+            dali.fn.stack(Z, shape_y, Z, Z),
+            dali.fn.stack(Z, Z, shape_x, Z),
+            dali.fn.stack(Z, Z, Z, shape_y)
+        )
+
+    T = dali.fn.stack(pos_x, pos_y, Z, Z)
+    return dali.fn.coord_transform(bboxes, M=M, T=T)
+
+def bbox_adjust_ltrb(bboxes, shape_x, shape_y, pos_x, pos_y):
+    sx, sy, ex, ey = pos_x, pos_y, shape_x + pos_x, shape_y + pos_y
+    MT = dali.fn.transforms.crop(
+        to_start=dali.fn.stack(sx, sy, sx, sy),
+        to_end=dali.fn.stack(ex, ey, ex, ey)
+    )
+    return dali.fn.coord_transform(bboxes, MT=MT)
+
+
+# Works for 2D tensors with equal second dimension size
+def select(bool_TL, if_true, if_false):
+    true_shape = dali.fn.cast(dali.fn.shapes(if_true), dtype=dali.types.DALIDataType.INT32)
+    false_shape = dali.fn.cast(dali.fn.shapes(if_false), dtype=dali.types.DALIDataType.INT32)
+
+    joined = dali.fn.cat(if_true, if_false)
+    sh = bool_TL * true_shape + (1 - bool_TL) * false_shape
+
+    st = dali.fn.cat(
+            dali.fn.slice(true_shape * (1 - bool_TL), start=[0], shape=[1], axes=[0]),
+            dali.fn.slice(true_shape * 0, start=[1], shape=[1], axes=[0])
+        )
+
+    return dali.fn.slice(joined, start=st, shape=sh, axes=[0,1], out_of_bounds_policy="trim_to_shape")
+
+
+def color_twist(images):
+    def random_value():
+        value = dali.fn.random.uniform(range=(1, 1.5))
+        coin = dali.fn.random.coin_flip()
+        return coin * value + (1.0 - coin) * (1.0 / value)
```

Comment:
Why not just `uniform(range=(2/3, 1.5))`? This approach produces kind of weird distribution where the interval (1.0, 1.5) is uniformly distributed and (2/3, 1.0) is not. Is it intended?